source/extensions/filters/network/thrift_proxy/BUILD

```diff
@@ -35,6 +35,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    category = "SOMECAT",
```

Comment:
i guess this is `envoy.filters.network`, but i also see

```
$ grep thrift catlist.txt 
envoy.thrift_proxy.filters
envoy.thrift_proxy.protocols
envoy.thrift_proxy.transports
```

so wondering if any of the thrift extensions are one of these

Reply:
That's right; the other extension points are for extension points that this filter itself declares.